packages/fluent-theme/src/components/preChatActivity/private/RecolorMonochromeImage.tsx

```diff
@@ -0,0 +1,19 @@
+import cx from 'classnames';
+import React, { memo, useMemo } from 'react';
+import { useStyles } from '../../../styles/index.js';
+import styles from './RecolorMonochromeImage.module.css';
+
+type Props = Readonly<{ className?: string | undefined; src: string }>;
+
+const RecolorMonochromeImage = ({ className, src }: Props) => {
+  const classNames = useStyles(styles);
+  const style = useMemo(() => ({ maskImage: `url(${src})` }), [src]);
+
+  return (
+    <div className={cx(className, classNames['pre-chat-message-activity__recolor-monochrome-image'])} style={style} />
```

Comment:
Suggesting using CSS variables for setting the image and updating styles to include `-webkit` prefixed value.

This way it is easier to reason about the change and we use `style=` only to set the variable rather adding styles dynamically.